organization/yearly-schedule.md

```diff
@@ -12,29 +12,23 @@ This schedule shows recurring events the Foundation for Public Code prepares for
 
 ## February
 
-1-2 February: [FOSDEM](https://fosdem.org/2020/) - see you there!
-
 ## March
 
-11-13 March: [Code for America Summit](https://www.codeforamerica.org/summit) - see you there!
-
 ## April
 
-2 April: Foundation for Public Code General Assembly
-
-13 April: Easter Monday (public holiday in the Netherlands, office closed)
+5 April: Easter Monday (public holiday in the Netherlands, office closed)
```

Comment:
Perhaps add that we're closed on Tuesday too?

Reply:
It's a exception / one-off so might be confusing. I'd prefer to leave it like this.